internal/test/e2e/run.go

```diff
@@ -204,19 +206,30 @@ func splitTests(testsList []string, conf ParallelRunConf) []instanceRunConf {
 
 	vsphereTestsRe := regexp.MustCompile(vsphereRegex)
 	privateNetworkTestsRe := regexp.MustCompile(`^.*(Proxy|RegistryMirror).*$`)
+	multiClusterTestsRe := regexp.MustCompile(`^.*Multicluster.*$`)
 
 	runConfs := make([]instanceRunConf, 0, conf.MaxInstances)
 	ipman := newE2EIPManager(os.Getenv(cidrVar), os.Getenv(privateNetworkCidrVar))
 
 	testsInCurrentInstance := make([]string, 0, testPerInstance)
 	for _, testName := range testsList {
 		testsInCurrentInstance = append(testsInCurrentInstance, testName)
-		ip := ""
+		multiClusterTest := multiClusterTestsRe.MatchString(testName)
+		var ips framework.E2EIPPool
 		if privateNetworkTestsRe.MatchString(testName) {
-			ip = ipman.getPrivateIP()
+			if multiClusterTest {
+				ips = ipman.reservePrivateIPPool(2)
+			} else {
+				ips = ipman.reservePrivateIPPool(1)
+			}
 		} else if vsphereTestsRe.MatchString(testName) {
-			ip = ipman.getIP()
+			if multiClusterTest {
+				ips = ipman.reserveIPPool(2)
```

Comment:
I would bump up this number, maybe 10

Reply:
Do we have a limit problems with the number of ips? Reserving an IP has no side effects here aside from that IP not being used by other tests. I thought the CIDR we had was more then big enough.

I was thinking that we'll probably won't ever have tests with more than 10 clusters, so 10 should cover most cases. If we keep it a 2, the moment a test adds more than one workload cluster, this won't work. And people will need to know this exist and come back to change it.